config/initializers/date_formats.rb

```diff
@@ -0,0 +1 @@
+Date::DATE_FORMATS[:mdy] = '%m/%d/%Y'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.